fetch.bs

```diff
@@ -7440,27 +7439,69 @@ fetch("https://www.example.com/")
 
 <h3 id=websocket-connections>Connections</h3>
 
+<p>A user agent has an associated <dfn>connecting WebSocket connections</dfn>, a <a for=/>map</a> of
+(<a for=/>network partition key</a>, <a for=/>host</a>, <a for=url>port</a>) to a
+<a for=/>set</a> of unique identifiers.
+
 <p>To <dfn id=concept-websocket-connection-obtain>obtain a WebSocket connection</dfn>, given a
-<var>url</var>, run these steps:
+<a>network partition key</a> <var>key</var>, <a for=/>origin</a> <var>origin</var>, and boolean
+<var>credentials</var>, run these steps:
 
 <ol>
- <li><p>Let <var ignore>host</var> be <var>url</var>'s <a for=url>host</a>.
+ <li><p>Let <var>hosts</var> be the result of running resolve an origin given <var>key</var> and
```

Comment:
Do we need language here to explicitly enable aborting the connection?

Reply:
This is a problem with #1245 as well. I can add a sentence there to make it clear they may be closed.